### ISSUE TITLE
Alignment of Trackname in TCP and the Clip-handle drag-bars

### ISSUE THREAD

**Issue by petersampsonaudacity** [audacity/audacity#1474]
### Discussed in https://github.com/audacity/audacity/discussions/1473

<div type='discussions-op-text'>

<sup>Originally posted by **petersampsonaudacity** August 13, 2021</sup>
**Describe the bug**
The trackname in Track Control Panel (TCP) is not aligned with the Clip-handle drag-bars (AKA "affordances")

**To Reproduce**
1) Get some audio
2) Observe the mis-alignment

**Expected behavior**
Either:
a) the height of the Clip-handle drag-bar to be reduced to the same size as the Track Name field in the TCP, or
b) the height of the Track Name field in the TCP to be enlarged to be the same height as the Clip-handle drag-bar

**Screenshots**
Image magnified  x 2.5
![image](https://user-images.githubusercontent.com/50205984/129359126-833506c3-7d4c-445c-9163-add5daa177c3.png)

**Additional information (please complete the following information):**
 - Windows 10 - but assume all OS
 - Audacity 3.1.0/3.0.4

**Additional context**
It would just look much neater and more visually/aesthetically pleasing if the two were in alignment.
</div>

**Comment by petersampsonaudacity** [comment]
I had to initiate this as a "Discussion" and then get it transferred using "Create issue from discussion" - as when I tried to just create the Issue the "Submit new issue" is faded out and inaccessible/un-clickable:

![image](https://user-images.githubusercontent.com/50205984/129361833-451b7b60-40c9-4d83-86f0-109d83962156.png)

**Comment by LWinterberg** [comment]
>  as when I tried to just create the Issue the "Submit new issue" is faded out and inaccessible/un-clickable

Was that because you didn't enter a title?

**Comment by petersampsonaudacity** [comment]
> 
> 
> > as when I tried to just create the Issue the "Submit new issue" is faded out and inaccessible/un-clickable
> 
> Was that because you didn't enter a title?

Yes Leo, embarrassingly so  _<blushes very deeply and face-palms>_

**Comment by petersampsonaudacity** [comment]
@jounih  wrote in the associated Issue #1473 

>18px for the clip handle bars sounds fine and looks like it would fix this. 
>@Paul-Licameli or @vsverchinsky could you please update kAffordancesAreaHeight to 18px? 
>And please send a link to the build to me/Martin once done, so we can check visually. 
>Thanks

**Comment by Paul-Licameli** [comment]
I see your screenshot and I wonder if you have been playing with the goniometer again.

**Comment by petersampsonaudacity** [comment]
> 
> 
> I see your screenshot and I wonder if you have been playing with the goniometer again.

Actually @Paul-Licameli I did in the past test the Goniometer with Sgt Pepper album  ;-)

**Comment by petersampsonaudacity** [comment]
This looks better now testing on W10 with audacity-win-3.0.4-alpha-20210817+7949a0d-64bit

Scaled  2.5x
![image](https://user-images.githubusercontent.com/50205984/129868265-35771b6d-a108-4fef-af39-a1c5576b9b1a.png)

**Comment by petersampsonaudacity** [comment]
Testing on W10 with latest alpha build audacity-win-3.0.4-alpha-20210820+45a90e3-64bit 

This tests good for this - the Clip-handle (and the box at the top of the Vertical Scale) is the same height and aligns nicely with the track name in the TCP

![image](https://user-images.githubusercontent.com/50205984/130264821-fef9e0ca-5caf-4378-a86b-e92ef43fcbc6.png)

And note that Audacity now just has the single Clip-handle drag-bar affordance for a stereo track
